services/py-api/src/utils.py

```diff
@@ -43,3 +49,26 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+ 
+    @staticmethod
+    def encode_data(data: Participant) -> str:
```

Comment:
```python
def encode_data(data: JwtUserData) -> str:
```
We don't want to encode the whole Participant obj into a JWT token, as it could have sensitive information, such as their email.

Reply:
Fixed in be223527926be992bf6729e80ee043728b8fa1c5